source/chapters/appendix.rst

```diff
@@ -305,19 +305,16 @@ Option                  Description
                         for user settings files such as the library
                         database and preferences configuration file.
 ----------------------  -------------------------------------------------
---controllerDebug       Causes Mixxx to display/log all controller data
-                        it sends and receives, and script functions it
-                        loads.
+--controllerDebug       Display/log all controller data Mixxx sends and
+                        receives as well as script functions it loads.
 ----------------------  -------------------------------------------------
---developer             Enables developer-mode. Includes extra log info,
-                        stats on performance, a Developer tools menu,
-                        extra developer skins, and provides useful
-                        information for skin writers in widget tooltips
-                        and logs .
+--developer             Enable developer-mode. Includes extra logs, stats
+                        on performance and the Developer tools menu as
+                        well as tooltips and logs useful for skin
+                        developers.
 ----------------------  -------------------------------------------------
---safeMode              Disables risky things at start up. This should
-                        help troubleshoot crashes at startup or if Mixxx
-                        won't boot.
+--safeMode              Disable risky things at start up. This should
+                        help troubleshoot startup crashes.
```

Comment:
Disable OpenGL widgets (scrolling waveforms, spinnies) to work around GPU driver bugs. If Mixxx is crashing on startup, try using this.

Reply:
Should I leave the list below that in?